packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -371,178 +376,119 @@ const PersonalDetailsForm = observer(() => {
-                                                        data-lpignore='true'
-                                                        type='text'
-                                                        id={'tax_identification_number'}
-                                                        name='tax_identification_number'
-                                                        label={localize('Tax identification number*')}
-                                                        //@ts-expect-error type of residence should not be null: needs to be updated in GetSettings type
-                                                        value={values.tax_identification_number}
-                                                        onChange={handleChange}
-                                                        onBlur={handleBlur}
-                                                        error={errors.tax_identification_number}
-                                                        disabled={isFieldDisabled('tax_identification_number')}
-                                                        required
-                                                    />
-                                                </fieldset>
-                                            )}
-                                            {'employment_status' in values && (
-                                                <fieldset className='account-form__fieldset'>
-                                                    {isDesktop ? (
-                                                        <Dropdown
-                                                            placeholder={localize('Employment status')}
-                                                            is_align_text_left
-                                                            name='employment_status'
-                                                            list={getEmploymentStatusList()}
-                                                            value={values.employment_status}
+                                {!is_virtual && (
+                                    <Fragment>
+                                        <FormSubHeader title={localize('Employment and tax information')} />
+                                        <EmploymentTaxDetailsContainer
+                                            editable_fields={employment_tax_editable_fields}
+                                            parent_ref={scroll_div_ref}
+                                            handleChange={mutate}
+                                            tin_validation_config={tin_validation_config}
+                                            is_tin_autoset={
+                                                account_settings.tax_identification_number === 'Approved000'
+                                            }
+                                        />
+                                        {!is_virtual && (
```

Comment:
why do we check here is_virtual if is is checked on line 379? :)

Reply:
Virtual account dosen't have this section. Hence hiding that